### PR TITLE
CLDC-275: Next incomplete section styling

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Toggle the highlight styling when the skip to next incomplete section is clicked

image

### CODE REVIEW

File: app/javascript/styles/_task-list.scss [hunk 1]
```diff
@@ -85,3 +85,9 @@
 //   display: block;
 //   border: 1px solid blue
 // }
+
+.app-task-list__item:target,
+.tasklist_item_highligh{
```
Comment (baarkerlounger) [inline app/javascript/styles/_task-list.scss]: highlight*?

File: app/javascript/packs/tasklist.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+window.addHighlight = function(elementId) {
+  document.getElementById(elementId).classList.add('tasklist_item_highligh');
```
Comment (baarkerlounger) [inline app/javascript/packs/tasklist.js]: I think for consistency it'll likely be neater and more manageable if we use Stimulus JS throughout to organise JS. Would basically just require adding a `tasklist_controller.js` with a `connect` or `initialize` function that does this and then attaching it as the `data-controller` html attribute on your <a> tag.

File: app/javascript/styles/_task-list.scss (hunk 1 above)
Comment (kosiakkatrina) [reply app/javascript/styles/_task-list.scss]: Nice!

File: app/javascript/packs/tasklist.js (hunk 2 above)
Comment (kosiakkatrina) [reply app/javascript/packs/tasklist.js]: Yup, can do. I'll wait for you to merge your PR first and will fix this

**Pull request closed without merging**